packages/core/docs/integrations/bazaarvoice.md

```diff
@@ -0,0 +1,129 @@
+# Bazaarvoice <Badge text="Enterprise" type="info" />
+
+::: warning Paid feature
+This feature is part of the Enterprise version. Please [contact our team](https://www.vuestorefront.io/contact/sales) if you'd like to use it in your project.
+:::
+
+## Introduction
+
+This package provides integration with [Bazaarvoice](https://www.bazaarvoice.com/).
+
+Bazaarvoice (BV) provides two SDKs to integrate with their platform - `bvapi.js` and `bv.js`. The latter is recommended because of the performance improvements and this is what is package uses under the hood.
+
+We recommend reading [this documentation page](https://knowledge.bazaarvoice.com/wp-content/conversations-prr/en_US/display/integrating_content_bv_js.html) before using the package.
+
+## Installation
+
+Install the required package:
+
+```sh
+yarn add @vsf-enterprise/bazaarvoice
+```
+
+Register its Nuxt module with the following configuration:
+
+```javascript
+// nuxt.config.js
+
+export default {
+  buildModules: [
+    [
+      '@vsf-enterprise/bazaarvoice/nuxt',
+      {
+        clientName: '',
+        siteId: '',
+        environment: '',
+        locale: ''
+      }
+    ],
+  ],
+  build: {
+    transpile: [
+      '@vsf-enterprise/bazaarvoice'
```

Comment:
why not in rawSources?

Reply:
In this case, it doesn't make a difference, but I changed it to be consistent with other docs